### ISSUE TITLE
Erroneous date format in RFC3164

### ISSUE THREAD

**Issue by loic-b** [pinojs/pino-syslog#6]
Hi, 

In file lib/rfc3164.js",  I see those lines:
```javascript
const zeroReg = /0/
const tstamp = moment(data.time).tz(options.tz).format('MMM DD HH:mm:ss').replace(zeroReg, ' ')
```
On Wed Jan 30 13:49:34 CET 2019, the tsstamp will be equal to
"Jan 3  12:43:12" instead of expected
"Jan 30 12:43:12"

The problem is that it replaces any zero in the string while the expected behavior for RFC3164 is to replaced only the leading zero of the day!

So, I guess it will also fail for month equals october...

**Comment by jsumners** [comment]
Thank you for the report. Are you able to submit a PR to fix it?

**Comment by loic-b** [comment]
I will try...

**Comment by jsumners** [comment]
Closed in #7.

Thank you @loic-b
